probe/listener.py

```diff
@@ -58,7 +64,9 @@ async def start(self):
                 #  https://bugs.python.org/issue29980
                 logger.error(f"{self}: Connection failed: {e}")
 
-            await asyncio.sleep(backoff_factor * attempts)
+            self.connected = False
+            backoff_time = backoff_factor * (2 ** (attempts - 1))
```

Comment:
Blir ikke dette litt heftig? Når `attempts = 15` blir det 8192 sekunder = over to timer?

Reply:
Hva kan være aktuell "cut-off"? I alle fall for kort sånn den er nå, men vet dessverre ikke om det skulle til 3 min eller 10 min. (ref. problemene torsdag, som så ut til å vare vesentlig lenger(?)) Så litt tilsiktet, men ikke sikker på at dette løser noe. (Om den mot formodning skulle klare å reconnected etter f.eks. 1 time, er det kanskje bedre enn ikke?)